src/it/resources/com/google/checkstyle/test/chapter5naming/rule53camelcase/InputCamelCaseDefined.java

```diff
@@ -59,26 +58,27 @@ void XMLHTTPRequest() {}
 
     class InnerBad {
 
-        int newCustomerID;
-        // violation above 'newCustomerID.* more than '1' .* capital letters.'
+      int newCustomerID;
+      // violation above 'newCustomerID.* more than '1' .* capital letters.'
 
-        boolean supportsIPv6OnIOS;
-        // violation above 'supportsIPv6OnIOS.* more than '1' .* capital letters.'
+      boolean supportsIPv6OnIOS;
+      // violation above 'supportsIPv6OnIOS.* more than '1' .* capital letters.'
 
-        void XMLHTTPRequest() {}
-        // violation above 'XMLHTTPRequest.* more than '1' .* capital letters.'
+      void XMLHTTPRequest() {}
+      // violation above 'XMLHTTPRequest.* more than '1' .* capital letters.'
     }
 
-        InputCamelCaseDefined anonymousBad
-            = new InputCamelCaseDefined() {
+    InputCamelCaseDefined anonymousBad =
+            new InputCamelCaseDefined() {
```

Comment:
this is correct update, `=` should be on end of line

Reply:
Yeah, Formatter did it